Makefile

```diff
@@ -51,15 +51,27 @@ check:
 test:
 	@python3 -m unittest discover testcase/unittest/ -p "test_*.py" || exit -1
 
-# clean-mq clean-build clean-pyc
-clean: clean-mq clean-build clean-pyc
+# Clean all: clean-process clean-mq clean-pyc clean-db clean-log
+clean: clean-process clean-mq clean-pyc clean-db clean-log
+
+linux-clean: clean-process clean-mq-linux clean-pyc clean-db clean-log
```

Comment:
you can check OS platform by 'uname'. check [this](https://stackoverflow.com/a/24563202).

Reply:
Applied. Thank you very much for the reference.